source/content/guides/wordpress-composer/pre-ga/02-wordpress-composer-managed.md

```diff
@@ -35,11 +35,11 @@ WordPress does not natively support [Composer](https://getcomposer.org/), howeve
 
 ## Create Your Site
 
-There are two ways you can spin up a site using the WordPress Composer Managed upstream:
+There are two ways you can spin up a site using the WordPress Composer Managed :
```

Comment:
removing `upstream` here makes this sentence make less sense "using _the_ WordPress Composer Managed" vs "using the WordPress Composer Managed _upstream_" because "upstream" is the noun. I'd re-add `upstream` or change to:

```suggestion
There are two ways you can spin up a site using WordPress Composer Managed :
```